jailer.go

```diff
@@ -395,18 +416,18 @@ func NewNaiveChrootStrategy(rootfs, kernelImagePath string) NaiveChrootStrategy
 	}
 }
 
-// ErrCreateMachineHandlerMissing occurs when the CreateMachineHandler is not
+// ErrJailerMissingHandler occurs when the CreateLogFilesHandlerName is not
 // present in FcInit.
-var ErrCreateMachineHandlerMissing = fmt.Errorf("%s is missing from FcInit's list", CreateMachineHandlerName)
+var ErrJailerMissingHandler = fmt.Errorf("%s is missing from FcInit's list", CreateLogFilesHandlerName)
 
 // AdaptHandlers will inject the LinkFilesHandler into the handler list.
 func (s NaiveChrootStrategy) AdaptHandlers(handlers *Handlers) error {
-	if !handlers.FcInit.Has(CreateMachineHandlerName) {
-		return ErrCreateMachineHandlerMissing
+	if !handlers.FcInit.Has(CreateLogFilesHandlerName) {
```

Comment:
What's the reason behind changing this function?

Reply:
Before it would link after create machine, however since fifos are created after create machine, we need to either update where the handler get's injected or rearrange the default handler list. I opted for updating where it get's injected as the list is ordered pretty logically.

Reply:
It would be good to add some contextual information into the godoc for this function.  "LinkFilesHandler must be called after CreateLogFilesHandler in order to ensure the fifos are linked properly".

Note that this *is* a breaking change, as we've changed the name of the symbol for the error.  Since we're pre-1.0 it's not impossible to change, but we should make the name flexible enough that we can change the message for the error without having to change the symbol again.  Something like `ErrRequiredHandlerMissing` would make more sense.

Reply:
Fixed

Reply:
@xibz Can you change this to `ErrRequiredHandlerMissing`?  `ErrRequiredMissingHandler` doesn't really seem grammatical to me.

Reply:
Np! fixed.